Dockerfile

```diff
@@ -19,9 +19,7 @@ RUN apk add --no-cache \
     php-mbstring \
     php-posix
 
-# installs latest fwd version
-RUN curl -L https://github.com/fireworkweb/fwd/blob/v1.0-rc.5/builds/fwd?raw=true -o /usr/local/bin/fwd
-RUN chmod +x /usr/local/bin/fwd
+COPY builds/fwd /usr/local/bin/fwd
```

Comment:
does it builds with executable permission?

Reply:
Yes, tested locally to make sure it had executable permissions

<img width="781" alt="image" src="https://user-images.githubusercontent.com/11093090/75066436-9862e880-54c9-11ea-9a3b-79e424bbba33.png">